views/support/faq.tt

```diff
@@ -0,0 +1,56 @@
+[%# faq.tt
+
+Support FAQ page
+
+Authors:
+    hotlevel4 <hotlevel4@hotmail.com>
+
+This program is free software; you may redistribute it and/or modify it under
+the same terms as Perl itself. For a copy of the license, please reference
+'perldoc perlartistic' or 'perldoc perlgpl'.
+
+%]
+
+[%- CALL dw.active_resource_group( "foundation" ) -%]
+
+[%- sections.title = '.title' | ml -%]
+
+<p>[% '.title.text2' | ml( sitenameshort = site.nameshort ) %]</p>
+<p>[% '.search.text' | ml %]</p>
+
+<form method='GET' action='[% site.root %]/support/faqsearch'>
+<div class="row">
+  <div class="large-12 columns">
+    <div class="row collapse">
+      <div class="small-10 columns">
+        <input type="text" placeholder="Search terms" name="q">
+      </div>
+      <div class="small-2 columns">
+        <input type='submit' class="button postfix" value='[% '.search.btn' | ml %]'>
+      </div>
+    </div>
+  </div>
+</div>    
+</form>
+
+
+<h2>[% '.toc.title' | ml %]</h2>
+<ol>
+    [% FOREACH faqcategory = faqcats %]
+        <li><a href = #[% faqcategory.faqcat %]>[% faqcategory.faqcatname %]</a></li>
+    [% END %]
+</ol><hr>
+
+[% FOREACH faqcategory = faqcats %]
+    <h2>[% faqcategory.faqcatname %] (<a href='faqbrowse?faqcat=[% faqcategory.faqcat %]' name='[% faqcategory.faqcat %]'>[% '.view.all' | ml %]</a>)</h2>
```

Comment:
we can do the ehtml here using the `| html` filter (since I suggested we remove it from the controller)